lib/config.dart

```diff
@@ -0,0 +1,3 @@
+library configs;
+
+const String FIREBASE_STORAGE_URI = 'gs://badiup.appspot.com/';
```

Comment:
Will `FIREBASE_STORAGE_URI` not be in lowerCamelCase?

Reply:
Good catch. Will fix this.